conda/environments/cudf_dev_cuda11.5.yml

```diff
@@ -38,15 +38,9 @@ dependencies:
   - ipython
   - pandoc<=2.0.0
   - cudatoolkit=11.5
-  - cuda-python >=11.5,<11.7.1
+  - cuda-python>=11.5,<11.7.1
   - pip
-  - flake8=3.8.3
```

Comment:
We're dropping these tools from the conda environment because they are all supposed to be handled by pre-commit.